### PR TITLE
feature: add footer menu

### PR DESCRIPTION
**Pull request by gcor**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -146,6 +146,40 @@
     </p>
   </main>
 
+  <div class="footer">
+    <a class="footerlogo" href="https://ekaterinburg.design/">
```
Comment (sashachabin) [inline index.html]: What about camelCase for BEM CSS classes for better long-word readability?

`footerLogo__image` :)

File: view/footer/index.css [hunk 2]
```diff
@@ -0,0 +1,120 @@
+.footer {
+    z-index: 999;
```
Comment (sashachabin) [inline view/footer/index.css]: Let's try to use small values (-1, 0, 1) or a little more? :)

File: view/footer/index.css [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+    font-weight: 400;
+    line-height: 1.1;
+}
+
+.footerlinks {
+    position: fixed;
+    display: flex;
+    bottom: 0;
+    right: 0;
+    z-index: 100;
```
Comment (sashachabin) [inline view/footer/index.css]: https://github.com/ekaterinburgdev/ekaterinburg.dev/pull/2/files#r1031763037

File: index.html (hunk 1 above)
Comment (gcor) [reply index.html]: done

File: view/footer/index.css (hunk 2 above)
Comment (gcor) [reply view/footer/index.css]: done)

File: view/footer/index.css (hunk 3 above)
Comment (gcor) [reply view/footer/index.css]: old

**Pull request merged**